### ISSUE TITLE
Multiple dovetail cuts

### ISSUE THREAD

**Issue by liftbag** [OrcaSlicer/OrcaSlicer#6976]
### Is there an existing issue for this feature request?

- [X] I have searched the existing issues

### Is your feature request related to a problem?

I recently helped a friend perform the cutting of an object with two dovetails, using a complicated procedure of cuts and boolean operations, as the current dovetail cut function does not allow for the insertion of more than one dovetail.

### Which printers will be beneficial to this feature?

All

### Describe the solution you'd like

I was thinking of implementing the option to insert at least a second dovetail, with its parameters, or even as a child of the first one, but with the ability to be positioned freely along the cutting axis.

### Describe alternatives you've considered

Multiple cut and sew operations only.

### Additional context

<img width="2035" alt="Screenshot 2024-09-30 alle 21 07 12" src="https://github.com/user-attachments/assets/7527e968-8d94-4482-9bff-78babb3abc98">

<img width="2031" alt="Screenshot 2024-09-30 alle 22 32 25" src="https://github.com/user-attachments/assets/36119518-9031-4a53-89fd-145eefe70604">

<img width="2031" alt="Screenshot 2024-09-30 alle 22 33 11" src="https://github.com/user-attachments/assets/9919127e-958f-4d8f-8cc5-ad80b8670114">

**Comment by paulohbsa** [comment]
I think the same.
Actually, I was a little bit frustrated when I realized it was not possible to do multiple dovetail cuts.

**Comment by cai876** [comment]
I agree, I would really like this feature too. One dovetail in not suitable for large parts

**Comment by pepies2137** [comment]
Found it searching for that function on google. Still a request guys. It would be really helpful :)

**Comment by dcchillin46** [comment]
Ya this would be handy

**Comment by RobertKoval** [comment]
up! it's weird that this feature wasn't implemented from the start.

**Comment by critter42** [comment]
Another vote for this feature - I have a very large box with thin walls that I need to cut to fit on the bed and need at LEAST 4 dovetails (two on the bottom and one each on the sides of the box at a minimum) to ensure alignment

**Comment by MawdleCode** [comment]
Went to make multiple Dovetails, found no such feature, found open request, added support, now sad.

**Comment by GracieSW584** [comment]
Also wanted multiple dovetails so I could make two parts out of a network rack shelf that is too big for my printer. Found open request.  Added support. Am now also sad.

**Comment by WingedRayeth** [comment]
Just want to add my vote for this. I'm currently needing a wide flat piece for my current project and I'd love the option to be able to do multiple dovetails to cut the piece in the way I need it.

**Comment by Robot-Coder2010** [comment]
I would also like this, but make sure they can be on different axis. Like one on the x and one on the y.

**Comment by LMaiorano** [comment]
+1 Another vote for this feature

**Comment by jbuszkie** [comment]
+1 I'd love to see this as well.

**Comment by carlinogianluca** [comment]
+1 ia a must have feuture

**Comment by Kasperinac** [comment]
This is exact what i need

**Comment by 0hCome0n** [comment]
As print beds become larger, this will become a more and more requested feature. I have a long thin box that needs to be cut with the slicer and _MUST_ have strength normal to the cut a single dovetail just isn't enough across 200 mm.

**Comment by ezric1** [comment]
Seems to me that instances along a plane should doable, similar to how Fusion operates. I would love this feature!

**Comment by liftbag** [comment]
I have the impression that an issue that has been open for 15 months is practically invisible to the developers. Between the bot that closes issues due to inactivity and the sheer number of issues opened every day, I can understand how this happens.
I was wondering if there is an alternative channel, possibly connected to GitHub, to help bring feature requests or long-standing issues to the developers’ attention.

**Comment by critter42** [comment]
> I have the impression that an issue that has been open for 15 months is practically invisible to the developers. Between the bot that closes issues due to inactivity and the sheer number of issues opened every day, I can understand how this happens. I was wondering if there is an alternative channel, possibly connected to GitHub, to help bring feature requests or long-standing issues to the developers’ attention.

I can think of two options - 1) code the feature yourself - it is open source, after all - and do a pull request (not trying to be flippant, I realize that most people that open Git issues aren't developers, but it's always an option!) 2) hit up Softfever on X or something and ask about this one.

**Comment by 0hCome0n** [comment]
It should be relatively easy for someone who knows what they're doing. I can't code it due to disability reasons, but whenever I think about why this feature could be difficult, I just run into "they would have to be insane to have coded it that way."

**Comment by Inspectorme13** [comment]
I assume that this has not yet been addressed. For those of you that Are still attempting to do this, here is a way. Its pretty simple and gets the job done until something like this gets its own feature, or as @critter42 stated, coding the gcode is also always an option.
https://forum.bambulab.com/t/multiple-dovetail-possible/135244
Anyhow, hope this helps

**Comment by Awatatah** [comment]
+1 this. crazy it doesn't already exist...

**Comment by liftbag** [comment]
> I assume that this has not yet been addressed. For those of you that Are still attempting to do this, here is a way. Its pretty simple and gets the job done until something like this gets its own feature, or as [@critter42](https://github.com/critter42) stated, coding the gcode is also always an option. https://forum.bambulab.com/t/multiple-dovetail-possible/135244 Anyhow, hope this helps


This is exactly what I did when, in the feature request description, I talked about the complicated cutting and boolean operations procedure. I only left out the export as STL and the subsequent re-import.

**Comment by ParadoxH88** [comment]
+1 pls boys!

**Comment by Xipit** [comment]
Hi, i am currently working on this. I have attached WIP screenshots.
Feel free to comment your specific usecases and what you might need from this feature. 

<img width="1073" height="693" alt="Image" src="https://github.com/user-attachments/assets/517a3095-05de-4204-92ce-926694b94847" />

<img width="1092" height="643" alt="Image" src="https://github.com/user-attachments/assets/070a52bf-b488-478d-ab02-4e3d84e2b9cd" />

**Comment by Felix14-v2** [comment]
@RF47 implemented, I guess?

**Comment by RF47** [comment]
Implemented in #12318
